setup/src/common/scripts/onCreateCommand.sh

```diff
@@ -21,6 +21,8 @@ sudo chown -R $(whoami) $HOME
 if [[ -z "${VELOCITAS_OFFLINE}" ]]; then
     .devcontainer/scripts/configure-codespaces.sh
     .devcontainer/scripts/upgrade-cli.sh
+elif [[ -x .devcontainer/scripts/local-setup.sh ]]; then
+    .devcontainer/scripts/local-setup.sh
```

Comment:
Is this change unrelated, or do we anywhere have a description on when/how/why you sometimes may use a `local-setup.sh` file and allowed content (like no actions requiring online connectivity?)

Reply:
It's related. With that I wanted to offer a possibility to do repo specific set-up for the offline variant. Problem is, that all the other scripts are "CLI maintained".

Reply:
Remark: I will probably change the location of this call. It's currently a bit strange from log output's perspective.